### DISCUSSION TITLE
Bonsai version history

### DISCUSSION THREAD

**Discussion by ZigaFiser** [bonsai-rx/bonsai#1462] [Q&A] [answered]
Hi,

Could anyone please refer me to a page/place where I can find Bonsai version history (with dates of publishing)? I am writing a methods parts for a study in which I used Bonsai, but as data gathering was done a few years ago, I already forgot which version was available at the time. 

Thanks and all the best!

**Comment by bruno-f-cruz** [comment]
Releases will likely be the closest you will get: https://github.com/bonsai-rx/bonsai/releases

Alternatively, consider using version control in the future: https://bonsai-rx.org/docs/articles/environments.html

**Reply by ZigaFiser** [reply]
Hi,

Thanks for a quick reply. I found the page with releases that you linked, but it goes back only to 2020. I am searching for a version that was available in October 2018. Huh, I know, long time ago. Are older releases also listed somewhere?

And thanks for the hint on using version control! I was not aware of this option and will definitely use it in my next Bonsai projects.

**Reply by bruno-f-cruz** [reply, marked as answer]
I guess you could check the [nuget feed](https://www.nuget.org/packages/Bonsai.Core#versions-body-tab). However, as in the releases, this will only list the version of your Bonsai.Core package. Do you have `bonsai.config` (located in the folder of the `bonsai.exe` executable) used for the experiments? All dependencies will be listed there.

**Reply by ZigaFiser** [reply]
Hi,

This list is exactly what I need and will suffice for my needs, thanks so much! I realize the info on dependencies is important, but I cannot go into so much details in the paper we are preparing. Thanks again!
